native/acter/src/api/room.rs

```diff
@@ -2367,6 +2390,34 @@ impl Room {
             })
             .await?
     }
+
+    pub async fn report_content(
+        &self,
+        event_id: String,
+        score: Option<i32>,
+        reason: Option<String>,
+    ) -> Result<bool> {
+        let room = if let SdkRoom::Joined(r) = &self.room {
+            r.clone()
+        } else {
+            bail!("Can't update power level in a room we are not in")
```

Comment:
this is probably not the right error message...

Reply:
Fixed